src/app/DoctrineMigrations/Version20180113214531.php

```diff
@@ -1,28 +0,0 @@
-<?php declare(strict_types = 1);
```

Comment:
Zmazanie tejto migracie je breaking change. Migracie mazat nemozes. Musis vytvorit novu, ktora ti databazu upravi tak aby to cele fungovalo dokopy.

Ak nevies ako, tak napis, poradim ti.

Reply:
neviem

Reply:
Večer napíšem ako. Zatiaľ vyrieš ostatné problémy

Reply:
1. Musim zmazat zmeny ktore si v tej tvojej migracii (novej, spravila).
```
php bin/console doctrine:migrations:migrate prev
```
Toto ti da databazu do stavu v akej bola predtym ako si zacala kodit.

2. Zmaz subor s tymi migraciami
```
rm  src/app/DoctrineMigrations/Version20180122200144.php
```

3. Applikuj vsetky migracie ktore boli na tvojej vetve (po rebasnuti).
```
php bin/console doctrine:migrations:migrate
```

Teraz by si mala mat databazu v takom stave ako by na mastrovi mala byt.

4. Vygeneruj migracie pre zmeny ktore si robila.
```
php bin/console doctrine:migrations:diff
```

5. Aplikuj dane migracie
```
php bin/console doctrine:migrations:migrate
```

Ak narazis na problem tak sa mi ozvi.